stixcore/tmtc/parameter.py

```diff
@@ -96,6 +98,46 @@ def flatten(self, root):
             self.value = [self.value]
             setattr(root, self.name, self)
 
+    def unpack_NIX00065(self):
+        """Unpack the NIX00065 values.
+
+        Continuation bits (NIXD0159) define number of subsequent bytes used to define counts for
+        given Detector / Pixel / Energy combination, i.e. value 0 denotes no following bytes and
+        count equal to 1, value 1 denotes 1 byte for “Counts” parameter with value between 2-255
+        and continuation bits equal to 2 are used for 2 successive bytes for “Counts” parameter
+        with value between 256 and 65535.
+
+        Parameters
+        ----------
+        param : ´Parameter´
+            The NIXD00159 parameter
+
+        Returns
+        -------
+        `Parameter`
+            The unpacked value
+
+        Raises
+        ------
+        ValueError
+            if unpacking schema is not supported
+        """
+        NIX00065 = None
+        if self.value == 0:
+            NIX00065 = 1
+        elif self.value == 1:
+            NIX00065 = self.children[0].value
+        elif self.value == 2:
+            high_bit, low_bit = [c.value for c in self.children]
+            NIX00065 = (high_bit << 8) + low_bit
+        else:
+            raise ValueError(f'Continuation bits value of {self.value} \
+            not allowed (0, 1, 2)')
+
+        param = Parameter(name=self.name, value=self.value, idb_info=self.idb_info,
+                          children=[Parameter(name='NIX00065', value=NIX00065, idb_info=None)])
+        return param
+
```

Comment:
i am not sure why this is marked as "new" this function was already available cince a long time